README.md

```diff
@@ -249,6 +249,11 @@ patch: |
 
 If set to `true`, the resulting k8s job spec is printed to the log. This can be useful when debugging.
 
+### `pod-backoff-limit` (optional, integer)
```

Comment:
```suggestion
### `job-backoff-limit` (optional, integer)
```
for consistency, let use job prefix similar to the ttlsecondsafterfinished since its a config on the job spec